client/login/wp-login/login-buttons.tsx

```diff
@@ -0,0 +1,134 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
Is there a reason to have this component only handling two buttons instead of all the ones available?

Google and Apple I guess are in another place and this handles Jetpack and Magic link?

Reply:
It may not be the best way, but in LoginButtons we will have all the additional buttons the we can add after the social ones already there 👍  In this way, the Social component doesn't know about these buttons and can be reused in Signup and in the old Login page as well